### PR TITLE
Docs: Add MDX examples for essentials

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

Add MDX examples for the essential addons, trying to make onboarding easier for MDX users, e.g. https://twitter.com/preshonyee/status/1319466429595398145

### CODE REVIEW

File: docs/essentials/actions.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ You can use [argTypes](../api/argtypes.md) to tell Storybook that an arg to your
 <CodeSnippets
   paths={[
     'common/button-story-onclick-action-argtype.js.mdx',
+    'common/button-story-onclick-action-argtype.mdx.mdx',
```
Comment (ndelangen) [inline docs/essentials/actions.md:27]: .mdx.mdx ?

Comment (ndelangen) [reply docs/essentials/actions.md:27]: just seems odd :)

**Review by ndelangen — APPROVED** [review]

File: docs/essentials/actions.md (hunk 1 above)
Comment (jonniebigodes) [reply docs/essentials/actions.md:27]: @ndelangen seems a bit meta i know :P ! But i like the pattern :D

**Review by jonniebigodes — APPROVED** [review]
Just ran through them and all good on my end @shilman . Thanks for the spacing fixes....Still have to figure out what's wrong with my mdx linter.

Good to merge.

**Pull request merged**